src/rpc/RPCHelpers.cpp

```diff
@@ -606,8 +606,10 @@ getLedgerInfoFromHashOrSeq(
         RPC::Status{RPC::RippledError::rpcLGR_NOT_FOUND, "ledgerNotFound"};
     if (ledgerHash)
     {
-        lgrInfo =
-            backend.fetchLedgerByHash(ripple::uint256{*ledgerHash}, yield);
+        // invoke uint256's constructor to parse the hex string , instead of
```

Comment:
This function is only called in new RPC framework. We should call uint256's string_view constructor. Pass string to construct uint256 will call its ctor overload, which treat the string as container and copy the buffer.
I also add the unittest for this.